library/src/amd_detail/rocblaslt/src/tensile_host.cpp

```diff
@@ -1146,23 +1117,34 @@ namespace
 } // namespace
 
 /******************************************************
-* Map a hipblas data type to a corresponding Tensile type *
-******************************************************/
-Tensile::DataType hipblasDatatype_to_tensile_type(hipblasDatatype_t type)
-{
-    switch(type)
-    {
-    case HIPBLAS_R_16F:
-        return Tensile::DataType::Half;
-    case HIPBLAS_R_32F:
-        return Tensile::DataType::Float;
-    case HIPBLAS_R_16B:
-        return Tensile::DataType::BFloat16;
-    default:
-        assert(!"hipblasDatatype_to_tensile_type: non-supported type");
-        return Tensile::DataType::None;
+ * Map a hipblaslt data type to a corresponding Tensile type *
+ ******************************************************/
+    Tensile::DataType hipblasltDatatype_to_tensile_type(hipblasltDatatype_t type)
```

Comment:
why did you remove the inline?

Reply:
Corrected, lost the specifier during editing.